lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional establishes what should happen when the code incounters a specific input. For example, a conditional could say if name == shannon, then that person is great.
```

Comment:
Other than checking if `x` is equal to `y`, what are two (or more) kinds of question you can ask in a conditional?